mm2src/coins/utxo/rpc_clients/electrum_rpc/client.rs

```diff
@@ -934,9 +939,24 @@ impl UtxoRpcClientOps for ElectrumClient {
         let mut hashes = vec![hex::encode(electrum_script_hash(&output_script))];
 
         // If the plain pubkey is available, fetch the balance found in P2PK output as well (if any).
+        // The output script (scriptPubkey) of such utxos may have the full pubkey length of 65 bytes or the compressed
+        // version of it (33 bytes). Both are valid and accepted by the blockchain.
```

Comment:
Do we handle it if the length is neither one? If not, should we?

Reply:
secp pubkeys are either 65 bytes (04 + x-coordinate + y-coordinate) point on the elliptic curve. or compressed 33 bytes (02 or 03 + x-coordinate).

other formats would be non-standard that no standard (non-modded) wallet supports.

Reply:
Yeah I know that, just asking if we have a sanity check to confirm that it fits in this standard.

Reply:
u mean what if the local pubkey we serializing is of different length?

u can check secp's `.serialize()` & `.serialize_uncompressed()` methods' return types. they return 33 bytes and 65 bytes resp.. and they are also infallible.